### PR TITLE
Fixed invalid selector on feature removal (Product page)

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | On feature(s) removal, all features were removed when we clicked on one of them. It's now fixed.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3889
| How to test?  | Try to delete a feature on Product Page before and after this patch.

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8436)

### CODE REVIEW

**Comment by LittleBigDev** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/prestashop/prestashop/8436#-:-KxDLchKSkYI-jEJghBJ:bnfp4nl)*

**Review by LittleBigDev — DISMISSED** [review]
Thanks @mickaelandrieu . Looks good to me.

**Comment by marionf** [comment]
@mickaelandrieu 
Same strange behavior as for the preview button.

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -390,7 +390,7 @@ var featuresCollection = (function() {
 
         modalConfirmation.create(translate_javascripts['Are you sure to delete this?'], null, {
           onContinue: function() {
-            _this.parent().parent().parent().remove();
+            _this.parent().parent().remove();
```
Comment (Quetzacoalt91) [inline admin-dev/themes/default/js/bundle/product/form.js]: `.parent().parent().parent().parent().parent().parent().parent().parent().parent()`.

Using a selector as a parameter of `parent()` could prevent another regression as soon as we update the DOM. Have you checked that is possible?

Comment (mickaelandrieu) [reply admin-dev/themes/default/js/bundle/product/form.js]: You're right => I've changed to `_this.closest('.product-feature').remove();` but GitHub doesnt seems to publish my modifications...

**Comment by LittleBigDev** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/prestashop/prestashop/8436)*

**Review by LittleBigDev — APPROVED** [review]
Commit finally passed.
LGTM again.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Okay for this PR.
However, did you see that adding a feature is impossible? Even if you see a successful save, refreshing the product make the feature disappear.

@vincentbz

**Pull request merged**